assets/js/data/swconf.js

```diff
@@ -22,6 +22,15 @@ const swconf = {
       {% endfor %}
     ],
 
+    {%- comment -%} The request url with below url part will not be cached. {%- endcomment -%}
+    denyUrls: [
+      {% for url in site.pwa.cache.deny_urls %}
+        {% unless url == empty %}
+          '{{ url }}'{%- unless forloop.last -%},{%- endunless -%}
+        {% endunless  %}
+      {% endfor %}
+    ],
```

Comment:
```suggestion
    {%- comment -%} URL requests that will not be cached by the PWA. {%- endcomment -%}
    denyUrls: [
      {% if site.analytics.goatcounter.id != nil and site.pageviews.provider == 'goatcounter' %}
        'https://{{ site.analytics.goatcounter.id }}.goatcounter.com/count'
      {% endif %}
    ],
```

Reply:
Applied suggested changes with more explicit comment.
Also modified comparison logic to `startsWith` in `sw.js`.